File: 02 Digits and Numbers - Foundation Programs/program-10.cpp
// Write a program that takes a 3-digit number as input and calculates the product of all its digits, then
// displays the result.


#include<iostream>
using namespace std;
int main()
{
    int number,first_digit,middle_digit,last_digit,product;
    cout<<"Enter 3-digit number : ";
    cin>>number;
    first_digit=number/100;
    middle_digit=(number/10)%10;
    last_digit=number%10;
    product=first_digit*middle_digit*last_digit;

    cout<<"The product of all its digits : "<<product;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-10.py
n = input("\nEnter a number : ")

try:
    n = int(n)
    if n < 1000 and n > 99:

        a = n//100
        b = (n//10)%10
        c = n%10

        print(f"Product of all digit : {a*b*c}\n")

    else:
        print("Enter a THREE Digit Number")
except ValueError:
    print("enter a valid number")